db/schema.rb

```diff
@@ -591,9 +592,9 @@
     t.boolean "no_cash_income"
     t.boolean "no_cash_outgoings"
     t.date "purgeable_on"
-    t.string "required_document_categories", default: [], null: false, array: true
     t.boolean "extra_employment_information"
     t.string "extra_employment_information_details"
+    t.string "required_document_categories", default: [], null: false, array: true
```

Comment:
this sneaked in during a rebase, but if others always find they get this change when applying a migration locally perhaps we could keep in